community/community.css

```diff
@@ -0,0 +1,65 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+header{
+    display: flex;
+    font-size: 1.3rem;
+}
+.prev{
+    color: #000000;
+    font-size: 2rem;
```

Comment:
Adding this code here looks more beautiful

`    margin-left:20px ;
    margin-top: -5px;
    `